source/_docs/migrate-drupal-gardens.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Migrating a Drupal Gardens Site to Pantheon
+description: Get all the details you need to know to successfully migrate your site away from Drupal Gardens.
+categories: [drupal]
+tags: [migrate]
+keywords: drupal, pantheon, drupal gardens, import
+---
+Drupal Gardens is ending support on August 1, 2016. This article walks you through exporting and migrating your site from Drupal Gardens to Pantheon.
+
+## Export a Site from Drupal Gardens
+
+1. Log in to your [Drupal Gardens account](https://www.drupalgardens.com/user/login).
+2. Go to **My Sites**.
+3. From the Actions menu next to the site you want to export, select **Export site**.
+4. Select "Want to host it on a 3rd party hosting company" as the reason for exporting, and enter an explanation.
+5. Click **Export**. This may take some time depending on the size of your site.
+6. Once it's finished exporting, save the file (.tar.gz archive) to your computer.
+
```

Comment:
Those apply to a different workflow. See this doc: https://my.bluehost.com/cgi/help/transfer-drupal-gardens